packages/ui-react/src/components/ChooseOfferForm/ChooseOfferForm.tsx

```diff
@@ -52,30 +61,26 @@ const OfferBox: React.FC<OfferBoxProps> = ({ offer, selected, onChange }: OfferB
         checked={selected}
         data-testid={testId(offer.offerId)}
       />
-      <label className={styles.label} htmlFor={offer.offerId}>
-        <h2 className={styles.offerTitle}>{title}</h2>
-        <hr className={styles.offerDivider} />
-        <ul className={styles.offerBenefits}>
-          {offer.freeDays || offer.freePeriods ? (
-            <li>
-              <Icon icon={CheckCircle} /> {getFreeTrialText(offer)}
-            </li>
-          ) : null}
-
-          {!!secondBenefit && (
-            <li>
-              <Icon icon={CheckCircle} /> {secondBenefit}
-            </li>
-          )}
-          <li>
-            <Icon icon={CheckCircle} /> {t('choose_offer.benefits.watch_on_all_devices')}
-          </li>
-        </ul>
-        <div className={styles.fill} />
-        <div className={styles.offerPrice}>
-          {getOfferPrice(offer)} {!!periodString && <small>/{periodString}</small>}
+      {/* @TODO: only when label is clicked, the styling such as white background gets applied */}
+      <div className={styles.offerDescription}>
+        <label htmlFor={offer.offerId}>
+          <h2 className={styles.offerTitle} id={`title-${offer.offerId}`}>
+            {title}
+          </h2>
+        </label>
```

Comment:
```suggestion
        <label htmlFor={offer.offerId} aria-describedby={`desc-${offer.offerId}`}>
          <h2 className={styles.offerTitle} id={`title-${offer.offerId}`}>
            {title}
          </h2>
        </label>
```

This approach was my original thought, in combination with the removal of `role="option"` in the div above.

But this solution also seems good (and quite similar) to me! Make sure to also test if keyboard navigation (arrow keys) still causes to switch the active offer. If not, maybe consider my approach.

I haven't tested on screen readers yet. Let me know if I need to do that. Since you are still in WIP phase, maybe it's to early to do that.